src/main/java/globalquake/ui/stationselect/StationSelectFrame.java

```diff
@@ -35,37 +75,77 @@ public void actionPerformed(ActionEvent actionEvent) {
             }
         });
 
-        add(stationSelectPanel, BorderLayout.CENTER);
-        add(createToolbar(), BorderLayout.PAGE_START);
+        JToolBar toolBar = createToolbar();
+        toggleButton.addActionListener(new AbstractAction() {
+            @Override
+            public void actionPerformed(ActionEvent actionEvent) {
+                if (toolBar.isVisible()) {
+                    toolBar.setVisible(false);
+                    toggleButton.setText(">");
+                } else {
+                    toolBar.setVisible(true);
+                    toggleButton.setText("<");
+                }
+            }
+        });
+        JPanel CenterPanel = new JPanel(new GridBagLayout());
```

Comment:
Please stick to the Java naming conventions and name the panel `centerPanel` instead of `CenterPanel`